### ISSUE TITLE
Level 1 (Homer) Spinning banner

### ISSUE THREAD

**Issue by f00god** [ZenoArrows/The-Simpsons-Hit-and-Run#21]
A spinning banner from a gas station flies away from its point towards Mr. Burns' estate.
![2023101601364000-B662483B2A4D01561FA37775965F956C](https://github.com/ZenoArrows/The-Simpsons-Hit-and-Run/assets/147378676/4718aef9-2e33-4ae6-b31c-d0f52671751a)

**Comment by ZenoArrows** [comment]
I was wondering when someone would notice, I found that really funny the first time I saw it fly off. I have no idea yet what causes it, it looks like it might be compiler optimizations.

**Comment by f00god** [comment]
I noticed this from the first day, but decided that this was not a significant problem to clutter the list with. The same problem, by the way, is also at the fourth level (Marge). There is a similar rotating sign next to this one, and everything is fine on that sign.

Is it possible to solve this problem with a banal ctrl+c ctrl+v? :D

**Comment by ZenoArrows** [comment]
Not really, I've compared the data from the two signs and they're exactly the same already.
